portfoliograph/graph.py

```diff
@@ -0,0 +1,133 @@
+from portfoliograph.hpd_regs import hpd_reg_where_clause
+from typing import Any, Dict, List, NamedTuple, Optional
+from enum import Enum
+import networkx as nx
+
+
+class NodeKind(Enum):
+    NAME = 1
+    BIZADDR = 2
+
+
+class Node(NamedTuple):
+    kind: NodeKind
+    name: str
+
+    def to_json(self) -> Dict[str, Any]:
+        # This format is an artifact from the way hpd-graph-fun's
+        # nodes were serialized. For more details, see:
+        #
+        #   https://github.com/JustFixNYC/hpd-graph-fun
+        if self.kind == NodeKind.NAME:
+            return {"Name": self.name}
```

Comment:
@toolness is there any reason you didn't add a "Type" property here so we can differentiate between names and bizaddr node types for easier visualization? And also make it easier to add new types

Reply:
Oh that's just because it's the default way a Rust library I used for hpd-graph-fun serialized enums.  I agree that it's not great--fixed in ad5ee4fd0cd6b5ec0b49abef301dfc26196924aa!